spec/components/prompts/input-spec.js

```diff
@@ -9,7 +9,7 @@ describe('InputPrompt', function () {
   var inputComponent;
```

Comment:
I know it wasn't introduced by you, but could you please replace `var` with `let`.